CakeMap.Rmd

```diff
@@ -95,6 +100,10 @@ difficult to provide general rules of thumb regarding simplicity other than
 'do not over-complicate the model with excessive constraint variables and
 constraints'.
 
+So, we always need to reach an equilibrium between these principles. 
```

Comment:
You start many sentences with "So, ". This sounds a little too informal to me...